lib/taiko.js

```diff
@@ -1654,7 +1654,7 @@ function addTaikoHighlightStyleIfNotPresent() {
 }
 
 /**
- * Clear all highlights for the current page.
+ * Clear all highlights marked using @highlight on the current page.
```

Comment:
Sorry - the syntax needs to use `@link` to link `highlight` properly. See https://jsdoc.app/tags-inline-link.html for example.

Currently this looks like this: https://deploy-preview-1146--ecstatic-raman-953525.netlify.com/#clearhighlights